tests/Integration/Functions/Options/rocketDeleteLicenceDataFile.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
The file should be located in `tests/Integration/Functions/`. `Options` would if it were a method on a class, where the class name is `Options`. 

Why?

All of the functions exist in the same namespace, which right now is the global namespace. They belong to the folder structure. Though they live in a particular file, they don't belong to that file per se. What do I mean? We could move the function to a different file without impacting its ability to run. Breaking it up into files helps us to organize common functionality around a single reason to exist. That's for organizational purposed.

A benefit of having all of the functions in the `Functions` folder is we can see a list of all functions to know the coverage.

Why are method test files within a folder named after the class? 2 reasons:

1. The method belongs to that class. It's bound to it. A function on the other hand is not. It exists outside of a class.
2. Name conflicts. Methods can have the same name across multiple classes. For example, we can have a method called `init` that exists within multiple classes. But we couldn't have `init.php` all within one folder for all of the classes in that folder. Right? So instead we put method test files within a folder named after its class. It bundles it together for that class.

Does that make sense?